### PR TITLE
LPS-121532 Fix margins and paddings

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by natocesarrego** [comment]
Hi @georgel-pop-lr, please use this last pull as a base to continue your work. 🙂

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_form-pagination.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@ $c: '.ddm-form-pagination';
 		min-height: 90px;
 	}
 
-	.dmm-wizard,
+	.ddm-wizard,
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_form-pagination.scss:27]: Nice job, reviewer @alinedoleron! 😉 😄

**Comment by natocesarrego** [comment]
One more thing, @georgel-pop-lr, I realized that the current implementation is displaying the message **Indicates required fields** on other product (App Builder) which shares code with Forms, please see the image below:

Screen Shot 2020-11-12 at 09 51 06

Please consider this issue during your implementation. If you need some assistance, please let us know. 🙂

**Comment by georgel-pop-lr** [comment]
> One more thing, @georgel-pop-lr, I realized that the current implementation is displaying the message **Indicates required fields** on other product (App Builder) which shares code with Forms, please see the image below:
> 
> Screen Shot 2020-11-12 at 09 51 06
> 
> Please consider this issue during your implementation. If you need some assistance, please let us know.

Hi @natocesarrego,

I should continue working on this pr? but not sure how to get the code from here, is possible someone to send me this pr? 

Regards.

**Comment by georgel-pop-lr** [comment]
> Hi @georgel-pop-lr, please use this last pull as a base to continue your work.

Hi @natocesarrego 

Could you please send me this pr ?

**Comment by georgel-pop-lr** [comment]
Just started reviewing :)

**Pull request closed without merging**